include/swift/AST/DiagnosticsParse.def

```diff
@@ -1485,6 +1485,10 @@ ERROR(duplicate_attribute,none,
       "duplicate %select{attribute|modifier}0", (bool))
 NOTE(previous_attribute,none,
      "%select{attribute|modifier}0 already specified here", (bool))
+ERROR(multiple_access_level_modifiers,none,
+      "multiple access-level modifiers specified", ())
+NOTE(previous_access_level_modifier,none,
+     "'%0' previously specified here", (StringRef))
```

Comment:
This is reminiscent of other "previously" notes like `multiple_override_prev` and reads as if `%0` was specified multiple times to me. I wish we could get away with just highlighting both modifiers, but our old diagnostics formatter can only highlight stuff on a single line. How about `previous modifier specified here`?

Reply:
Your suggestion looks good to me, but considering your previous suggestion of adding the word "incompatible" to the error message, what do you think of the following?
`incompatible modifier specified here`

Reply:
> what do you think of the following?
> `incompatible modifier specified here`

I incline to the neutral variant. "incompatible" here makes the situation sound like a one-way problem, like the previous attribute is "the bad guy".

Reply:
Got it!